Fabric.Identity.IntegrationTests/IntegrationTestsFixture.cs

```diff
@@ -229,6 +306,89 @@ private static IS4.ApiResource GetTestApiResource()
+            var file = new FileInfo("Fabric.Identity.SqlServer_Create.sql");
+            var createDbScript = file.OpenText().ReadToEnd()
+                .Replace("$(DatabaseName)", $"Identity-{DatabaseNameSuffix}");
+
+            var splitter = new[] { "GO\r\n" };
+            var commandTexts = createDbScript.Split(splitter, StringSplitOptions.RemoveEmptyEntries);
+
+            int x;
+            using (var conn = new SqlConnection(connection))
+            {
+                conn.Open();
+                using (var command = new SqlCommand("query", conn))
+                {
+                    for (x = 0; x < commandTexts.Length; x++)
+                    {
+                        var commandText = commandTexts[x];
+
+                        // break if we just created the Identity DB
+                        if (commandText.StartsWith("CREATE DATABASE"))
+                        {
+                            command.CommandText = commandText.TrimEnd(Environment.NewLine.ToCharArray());
+                            command.ExecuteNonQuery();
+                            break;
+                        }
+                    }
+                }
+            }
+
+            // establish a connection to the newly created Identity DB
+            using (var conn = new SqlConnection(ConnectionStrings.IdentityDatabase))
+            {
+                conn.Open();
+
+                using (var command = new SqlCommand("query", conn))
+                {
+                    for (x = x + 1; x < commandTexts.Length; x++)
+                    {
+                        var commandText = commandTexts[x];
+
+                        // skip generated SqlPackage commands and comments
+                        if (commandText.StartsWith(":") || commandText.StartsWith("/*"))
+                        {
+                            continue;
+                        }
+
+                        command.CommandText = commandText.TrimEnd(Environment.NewLine.ToCharArray());
+
+                        try
+                        {
+                            command.ExecuteNonQuery();
+                        }
+                        catch (Exception ex)
+                        {
+                            Console.WriteLine(ex);
+                        }
+                    }
+                }
+            }
+        }
+
+        private static void AddTestEntitiesToSql(IS4.Client client, IS4.ApiResource apiResource)
+        {
+            using (var identityContext = IdentityDbContext)
+            {
+                identityContext.Database.ExecuteSqlCommand(DeleteDataSql);
+                identityContext.ApiResources.Add(apiResource.ToEntity());
+                identityContext.Clients.Add(client.ToEntity());
+                identityContext.SaveChanges();
+            }
+        }
+
+        private static readonly string DeleteDataSql =

```

Comment:
i dont think we need to run this anymore if we are creating a new DB for each test run